### PR TITLE
chore: update TypeScript

### PR DESCRIPTION
**Pull request by chrismwendt**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7307,9 +7307,9 @@ typedarray@^0.0.6:
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
 typescript@^3.2.1:
```
Comment (felixfbecker) [inline yarn.lock]: This looks wrong?

Comment (chrismwendt) [reply yarn.lock]: Oh yup, fixed. `yarn upgrade` didn't update `package.json`, then I bumped it manually.

File: src/positions.ts [hunk 2]
```diff
@@ -57,7 +57,10 @@ export const findPositionsFromEvents = (options: DOMFunctions) => (
             map(event => ({ event, eventType: 'click' as 'click' })),
             // ignore click events caused by the user selecting text.
             // Selecting text should not mess with the hover, hover pinning nor the URL.
-            filter(() => window.getSelection().toString() === ''),
+            filter(() => {
+                const selection = window.getSelection()
+                return selection === null || selection.toString() === ''
+            }),
```
Comment (chrismwendt) [inline src/positions.ts]: @felixfbecker Does this look correct to you? `window.getSelection(): Selection | null` (it's nullable) in TS 3.5

Comment (chrismwendt) [reply src/positions.ts]: Perhaps `Boolean(window.getSelection())` works just as well, and is more concise.

**Pull request merged**